runtime/compiler/env/CHTable.cpp

```diff
@@ -250,6 +250,16 @@ void TR_CHTable::cleanupNewlyExtendedInfo(TR::Compilation *comp)
       }
    }
 
+
+bool TR_CHTable::canSkipCommit(TR::Compilation *comp)
+   {
+   return comp->fej9()->isAOT_DEPRECATED_DO_NOT_USE() ||
```

Comment:
I don't know if it's worth creating a whole new FE query to represent "isAOT", but I suppose at the very least it's cleaner to use `comp->compileRelocatableCode()`.

Reply:
Code updated with the suggestion